lib/api.ts

```
@@ -87,94 +110,155 @@ export async function fetchScreenshots(
+
+  // Parse the CSV-like format into structured data
+  const soundData = [];
+  const lines = text.trim().split("\n");
+
+  for (const line of lines) {
+    if (line.trim()) {
+      const [timestampStr, levelStr] = line.split(",");
+      if (timestampStr && levelStr) {
+        soundData.push({
+          timestamp: timestampStr,
+          level: parseFloat(levelStr),
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)